README.md

```diff
@@ -1,2 +1,27 @@
-# fe-interview-design-system
-Public repository for the home test and pair programming exercise
+# Frontend Interview - Design System
+
+Hey 👋
+
+This is the base repository for the home test. The repository is created with `vite` and is empty, but contains some packages already installed, in particular:
+
+- `react`
+- `storybook`
+- `vitest`
+
+## Install and run
+
+```bash
+# Install dependencies
+# You can use also `npm` or `yarn`
```

Comment:
i'll put it like this
# Install dependencies 
this project use pnpm as package manager, but if you not have it you can alternatively use `npm` or `yarn`